src/Content/Conversation.php

```diff
@@ -784,7 +784,7 @@ public function create(array $items, string $mode, $update, bool $preview = fals
 		}
 
 		$o = Renderer::replaceMacros($page_template, [
-			'$baseurl'     => $this->baseURL->get($ssl_state),
```

Comment:
No, this was necessary in case we use SSL, but don't force it. With this flag we could enforce SSL for some links in these circumstances. But now the baseurl is either with or without SSL at all.

Reply:
Yes, and what about the variable you removed a reference to, is it still used elsewhere?